discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -80,7 +80,7 @@ def update_track_price_history(
     track_record: Track,
     track_metadata,
     blocknumber: int,
-    block_datetime: datetime,
+    timestamp: datetime,
```

Comment:
Wondering if this should use object arg at this point

Reply:
we can if needed later